doc/github_actions.rst

```diff
@@ -0,0 +1,39 @@
+Integration with GitHub Actions
+===============================
+
+`Universum` requires no special integration with `GitHub Actions <https://docs.github.com/en/actions>`_. It is usually
+launched as one long step in a single build stage.
+
+.. warning::
+
+    GitHub Actions doesn't support `report_artifacts` key in configuration. If specified, a broken link will be created.
+
+Command line
+------------
+
+To use `universum` to check-out current repository (the repository that contains the GitHub Action workflow) is
```

Comment:
"To use <...> is necessary to use" is grammatically incorrect.
Please rephrase, e.g. "Doing smth requires doing smth else"